package.json

```diff
@@ -3,10 +3,10 @@
   "private": true,
   "version": "1.0.0",
   "scripts": {
-    "postinstall": "cd ui5-service-worker && npm install && cd ../openui5-sample-app && npm install",
+    "build-ui5": "cd openui5-sample-app && npm run build",
+    "build-h2": "cd openui5-sample-app && npm run build --h2",
```

Comment:
The `build` command doesn't have a `--h2` flag. The `start-h2` script looks correct to me.

Reply:
True :-). I've adjusted the commands